src/components/Login/Login.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import { useAppContext } from '../../AppContext';
+
+import { Button } from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+
+import './Login.css';
+
+const baseUrl = 'http://localhost:3000';
+
+export default function Login() {
+	const { setUserName } = useAppContext();
+	const [userEmail, setUserEmail] = useState('');
+	const [userPassword, setUserPassword] = useState('');
+	const { setIsAuthenticated } = useAppContext();
+	const history = useHistory();
+
+	function handleLogin(e) {
+		e.preventDefault();
+
+		const data = { email: userEmail, password: userPassword };
+
+		axios
+			.post(`${baseUrl}/login`, data)
+			.then((response) => {
+				const authKey = response.data.result.split(' ')[1];
+				console.log(authKey);
+				setIsAuthenticated(true);
+				localStorage.setItem('authKey', authKey);
+				setUserName(response.data.user.name);
+				history.push('/courses');
+			})
+			.catch((e) => alert(e.response.data.errors.join('\n')));
+	}
+
+	return (
+		<div className='container'>
+			<form className='form'>
+				<h2>Login</h2>
+				<Input
+					labelText='Email'
+					labelClassName='label'
```

Comment:
Understood. Made label class more precise for now.

Reply:
Sooo the idea is to avoid lot's of classes for the same thing, this is login form but you see that Registration.css overwrides login styles. My suggestion was to do something like this:
![image](https://user-images.githubusercontent.com/28926420/153390671-9aebb0b5-28af-42a6-9a33-dcea58fdee4f.png)
You add general class here, remove styles from Login and Registraton and put styles in Input.css because it can be used not only on Registration and Login page. As a result there's no overwriding 
![image](https://user-images.githubusercontent.com/28926420/153390912-28aac9c5-c51b-44fa-b2f7-98640de22c8f.png)
The same for label. After that you have no extra css

Reply:
Right. Changed it :)